test/drivers/test_suites/syscall_exit_suite/open_by_handle_at_x.cpp

```diff
@@ -187,11 +189,14 @@ TEST(SyscallExit, open_by_handle_atX_success)
 	/* Parameter 4: path (type: PT_FSPATH) */
 	evt_test->assert_charbuf_param(4, fspath);
 
+#if defined(__loongarch64)
+#elif
```

Comment:
Out of curiosity, can you share the errors you were having here? 
We are having a very similar issue in this test case for ppc64le: https://github.com/falcosecurity/libs/pull/1739

Reply:
Also, i'd use `ifndef` in this case and leave a comment explaining the issues on the arch ;)

Reply:
I'm adding a judgment here because there is no fstat syscall call under the loongarch architecture. Let's take a look at the error situation without adding a judgment

Reply:
@FedeDP A compile error occurred because fstat could not be found.